vllm/entrypoints/openai/api_server.py

```diff
@@ -57,7 +57,7 @@
 openai_serving_tokenization: OpenAIServingTokenization
 prometheus_multiproc_dir: tempfile.TemporaryDirectory
 
-logger = init_logger('vllm.entrypoints.openai.api_server')
+logger = init_logger(__name__)
```

Comment:
Does this not reintroduce the issue fixed by #4765?

Reply:
will revert

Reply:
would be great to put a comment there too so others don't do the same thing!